src/components/molecules/TableComponent/TableComponent.module.scss

```diff
@@ -0,0 +1,80 @@
+@use "scss/attendee/space";
+@use "scss/attendee/effects/sparkle";
+@use "scss/attendee/effects/rounded";
+@use "scss/attendee/effects/atom";
+@use "scss/attendee/theme";
+@use "scss/attendee/breakpoint";
+
+.tableContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+
+  margin: space.gap(3);
+  flex-direction: column;
+  @include sparkle.panel--glass;
+  @include rounded.normal;
+  border: 1px solid theme.$sparkle--border;
```

Comment:
[OSS]
If enough of these get repeated often enough, it might be useful to not just use `border$width--slim` but have the style and color as well merged into few reusable SCSS options